app/classifier/drawing-tools/point-grid.jsx

```diff
@@ -0,0 +1,101 @@
+const React = require('react');
+const DrawingToolRoot = require('./root');
+const deleteIfOutOfBounds = require('./delete-if-out-of-bounds');
+const isInBounds = require('../../lib/is-in-bounds');
+const DeleteButton = require('./delete-button');
+
+const DELETE_BUTTON_ANGLE = 45;
+const SELECTED_RADIUS = {
+  large: 20,
+  small: 10
+};
+
+class PointGridTool extends React.Component {
+
+  static defaultValues({x, y}) {
+    return {x, y};
+  }
+
+  static initStart({x, y}) {
+    return {x: x, y: y, _inProgress: true};
+  }
+
+  static initRelease() {
+    return {_inProgress: false}
+  }
+
+  static initValid(mark, props) {
+    const naturalWidth = props.naturalWidth
+    const naturalHeight = props.naturalHeight
+    const offset_x = Number.parseInt(props.task.tools[mark.tool].x_offset)
+    const offset_y = Number.parseInt(props.task.tools[mark.tool].y_offset)
+
+    const notBeyondWidth = mark.x < naturalWidth;
+    const notBeyondHeight = mark.y < naturalHeight;
+    const beyondOffset = mark.x > offset_x && mark.y > offset_y
+
+    return notBeyondWidth && notBeyondHeight && beyondOffset;
+  }
+
+  static options = ['grid']
```

Comment:
I think we can remove that rule. I’ll double check tomorrow, but I remember the PR that removed `createClass` added a Babel plugin for `static`, because the codemod used it.

Reply:
You're right. It looks like this babel plugin was added: https://babeljs.io/docs/plugins/transform-class-properties/

We just need to get ESLint rule set updated because it's flagging this as an error.